package.json

```diff
@@ -63,6 +63,7 @@
     "jest-junit": "12.0.0",
     "lerna": "^3.22.1",
     "lint-staged": "^10.5.0",
+    "patch-package": "^6.1.2",
```

Comment:
Yarn has builtin support for patching packages using the `patch` protocol https://yarnpkg.com/features/protocols/#patch